pkg/liquidity_provider.go

```
@@ -148,6 +149,35 @@ func FromPegoutConfigurationDTO(dto PegoutConfigurationDTO) liquidity_provider.P
 	}
 }
 
+func ToPeginConfigurationDTO(config liquidity_provider.PeginConfiguration) PeginConfigurationDTO {
+	feePercentage, _ := config.FeePercentage.Native().Float64()
```

Should do error handling here, avoid ignore errors.